src/main/webapp/app/shared/layout/header/header.tsx

```diff
@@ -16,6 +18,7 @@ export interface IHeaderProps {
   isSwaggerEnabled: boolean;
   currentLocale: string;
   onLocaleChange: Function;
+  isCanvas: boolean;
 }
 
 const Header = (props: IHeaderProps) => {
```

Comment:
this should be a functional component

Reply:
I have not created this component
I have just added one key in props